src/styles/patterns/dswp-large-image-text.scss

```diff
@@ -0,0 +1,49 @@
+@media (max-width: $break-medium) {
+
+	.dswp-large-image-text {
+
+		img {
+			width: 100% !important;
+		}
+	}
+
+}
+
+@media (min-width: $break-mobile) and (max-width: $break-medium) {
```

Comment:
These could be refactored into a nested structure to remove a little bit of repetition.